config/production.json

```diff
@@ -15,12 +15,12 @@
   },
   "log": {
     "app": {
-      "level": "debug",
+      "level": "info",
       "stream": "stdout"
     },
     "process": {
-      "level": "debug",
-      "stream": "logs/process.log"
+      "level": "info",
+      "stream": "stdout"
```

Comment:
Stream logs to stdout, since Elastic Beanstalk's nodejs wrapper does a perfectly nice job of collecting and rotating them.